.github/workflows/ci-build.yml

```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches:
       - master
-      - 'v*'
```

Comment:
we're not releasing on tag-pushes as we have a mass migration to release-plan which tags as a part of releasing, rather than releasing because of a tag